### ISSUE TITLE
libraries/tutorials/etc device type labels

### ISSUE THREAD

**Issue by tripflex** [tripflex/awesome-mongoose-os#2]
Would be good to probably have a label/tag or something we can attach or place next to any links to specify if it's specific to a device (like esp32, etc)

**Comment by mamuesp** [comment]
Yes, badges would be nice, and perhaps a badge to tag the state of the lib (prod/late beta/early beta/alpha/in development)
